docs/configuration.rst

```diff
@@ -654,12 +660,20 @@ Up to infrasim-compute commit `a02417c3 <https://github.com/InfraSIM/infrasim-co
 
 .. _yamlComputeStoragebackendControllerDrivesSize:
 
+- **compute:storage_backend:-:controller:drives:-:page-file**
+
+    This option allows user to specify drive page data, which can provide addtional information for client OS.
+    command, e.g. ``generate_page_data.py -d /dev/sdb -o drive_name.bin``, will be called to create such a drive page bin file.
+    For more details, please refer to: <link>
```

Comment:
the python script is still in unpublished state.  we will make the link first, then publish the tool.